Sources/App/Settings/SettingsRootDataSource.swift

```diff
@@ -95,6 +97,16 @@ enum SettingsRootDataSource {
         }
     }
 
+    private static func security() -> SettingsButtonRow {
+        SettingsButtonRow {
```

Comment:
how does this work on catalyst? should we hide it or make it available -- if the latter, how do we deal with e.g. the settings scene?

Reply:
In catalyst it was a bit trickier, because of the "enter background/foreground" thats happens all the time even with the touchID dialog, I treated it with an extra logic to only reauthenticate when the user has reply enter background.

When the user open settings scene, the main web view scene locks again until the user unlocks it.

https://github.com/home-assistant/iOS/assets/5808343/7fdb056c-5339-476a-be25-f55dbec457cd

https://github.com/home-assistant/iOS/assets/5808343/6a0292af-024d-40d2-8899-059da3c3f62e